controller/internal/enforcer/applicationproxy/http/http.go

```diff
@@ -270,6 +270,8 @@ func (p *Config) processAppRequest(w http.ResponseWriter, r *http.Request) {
 	_, netaction, noNetAccesPolicy := puContext.ApplicationACLPolicyFromAddr(originalDestination.IP.To4(), uint16(originalDestination.Port))
 	if noNetAccesPolicy == nil && netaction.Action.Rejected() {
 		http.Error(w, fmt.Sprintf("Unauthorized Service - Rejected Outgoing Request by Network Policies"), http.StatusNetworkAuthenticationRequired)
+		record.Destination.ID = netaction.ServiceID
+		record.PolicyID = netaction.PolicyID
 		p.collector.CollectFlowEvent(record)
 		return
```

Comment:
Actually if you move this above the if, it will populated for accepted flows as well.

Reply:
done.